src/XMakeTasks/CommandLineBuilderExtension.cs

```diff
@@ -14,12 +14,7 @@ namespace Microsoft.Build.Tasks
     /// <summary>
     /// CommandLineBuilder derived class for specialized logic specific to MSBuild tasks
     /// </summary>
-#if WHIDBEY_VISIBILITY
-    internal
-#else
-    public /* because its used by VJ# Vjc task. */
```

Comment:
I am not sure comment is right. The class is being used as a parameter in public methods. Making it internal may be possible, but would require more extensive changes that just this file.